setup.py

```diff
@@ -41,7 +41,7 @@
 
   install_requires=[
     'matplotlib',
-    'networkx',
+    'networkx==1.11',
```

Comment:
this might be too narrow. persumably `networkx~=1.0` is fine

Reply:
Already done https://github.com/ginkgobioworks/pychemy/pull/5/commits/90c51dabb6896b6cdefef56a75e6bc034305495b